### ISSUE TITLE
v 2.0

### ISSUE THREAD

**Issue by zoutepopcorn** [things4u/ESP-1ch-Gateway-v2.0#1]
@platenspeler can you please point from the old repo README to the most recent Repo. Because there are 3 versions now i think 

I think this is the latest, working for me :+1: . But I always google for it so I didnt know you had a new Repo :D...

**Comment by platenspeler** [comment]
Good point, do you mean you like to see a complete new fork for every major release? I'm not a sourceTree savvy person, so I like to do as little as possible in order to not brake things :-)

I do have another release ready where you can use WiFiManager and the SSID array at the same time. So making a connection over WiFi either by giving the SSID of by using WiFiManager should be ease.

btw It also stores the value in SPIFF filesystem (still testing).

I will delete pone of the versions on github now because it is empty...

**Comment by zoutepopcorn** [comment]
Great update :). I think adding a link  to the Old readme pointing to the latest Version. And using a wat to update you want ;)

I was using the Old one for a while because i didn't  discovered the New one.
